{"name": "test_true", "status": "failed", "statusDetails": {"message": "AssertionError: assert False == True\n  +False\n  -True", "trace": "self = <test_leap_year.TestAssert object at 0x0000027505D4C9C8>\n\n    def test_true(self):\n>       assert is_leap_year(2017) == True\nE       assert False == True\nE         +False\nE         -True\n\ntest_leap_year.py:33: AssertionError"}, "start": 1588052994919, "stop": 1588052994919, "uuid": "c8d8447b-4b3a-4cec-9214-d8979c671c87", "historyId": "9db7ba8602f272ed967dcdafbb3f3521", "testCaseId": "34603d2208cc04ae1d25f5b0c05a0fa3", "fullName": "test_leap_year.TestAssert#test_true", "labels": [{"name": "suite", "value": "test_leap_year"}, {"name": "subSuite", "value": "TestAssert"}, {"name": "host", "value": "PC-BFD8TY2"}, {"name": "thread", "value": "10460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_leap_year"}]}